src/blocklyc.js

```diff
@@ -673,9 +646,9 @@ function loadInto(modal_message, compile_command, load_option, load_action) {
                         }
                     });
                 } else {
-                    //TODO: Remove this once client_min_version is >= minCodedVer
+                    //TODO: Remove this once clientService.version.isCoded is no longer used
                     //Request load without options from old BlocklyProp Client
-                    $.post("http://" + client_domain_name + ":" + client_domain_port + "/load.action", {
+                    $.post(clientService.url('http') + 'load.action', {
```

Comment:
`clientService` contains a `url()` function that takes an argument ('ws' or 'http'), and combines it with the path and port for the client and returns a formed url string.